test/storage/file.js

```diff
@@ -281,6 +281,126 @@ describe('File', function() {
     });
   });
 
+  describe('move', function() {
+    it('should throw if no destination is provided', function() {
+      assert.throws(function() {
+        file.move();
+      }, /should have a name/);
+    });
+
+    describe('destination types', function() {
```

Comment:
It's hard to give a negative review of the tests, because they're great at testing the functionality you added. But, we tend to test a bit differently in the library, making use of stubs wherever possible, not caring about if the methods we use do their job properly, as we have other tests for that.

As an example of how we would test this change, we would stub the "copy" method, and see that it was called with the correct arguments. We would then see that our callback was executed with the response we mocked. We also check for errors wherever they can occur, and make sure the provided callback gets them.

I can elaborate with some examples tomorrow, and as always, you're free to leave the PR as-is and let me worry about it :)

Thanks again for the contribution!